README.md

```diff
@@ -1,94 +1,91 @@
+We welcome contributions from collaborators and researchers interested in our work. If you have improvements, suggestions, or new findings to share, please submit a pull request. Your contributions help advance our research and analysis efforts.
 
-Most dependencies are optional installations for this package. 
-Below are examples of various installation configurations.
+To get started with your development (or fork), click the "Open with GitHub Codespaces" button below to launch a fully configured development environment with all the necessary tools and extensions.
 
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/braingeneers/braingeneerspy?quickstart=1)
+
+Instruction on how to contribute to this project can be found in the CONTRIBUTION.md
+
+## Installation
+
+You can install `braingeneerspy` using `pip` with the following commands:
+
+### Install from PyPI (Recommended)
+
+```bash
+pip install braingeneerspy
 ```
-# Typical install (includes `iot`, `analysis`, and `data` access functions, skips `ml`, and lab-specific dependencies): 
-python -m pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[iot,analysis,data]
 
-# Full install (all optional dependencies included).
-python -m pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[all]
+### Install from GitHub
 
-# Minimum install (no optional dependencies, good for Raspberry PI builds).
-python -m pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git
+```bash
+pip install --force-reinstall git+https://github.com/braingeneers/braingeneerspy.git
 ```
 
-### macOS installation note:
-if install fails with ```no matches found: git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[all]```
-wrap quotes around the github address like so 
+### Install with Optional Dependencies
+
+You can install `braingeneerspy` with specific optional dependencies based on your needs. Use the following command examples:
 
+- Install with IoT, analysis, and data access functions (skips machine learning and lab-specific dependencies):
+
+```bash
+pip install "braingeneers[iot,analysis,data]"
 ```
-# Typical install (includes `iot`, `analysis`, and `data` access functions, skips `ml`, and lab-specific dependencies): 
-python -m pip install --force-reinstall 'git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[iot,analysis]'
 
-# Full install (all optional dependencies included).
-python -m pip install --force-reinstall 'git+https://github.com/braingeneers/braingeneerspy.git#egg=braingeneerspy[all]'
+- Install with all optional dependencies:
+
+```bash
+pip install "braingeneers[all]"
 ```
 
-### Optional dependency organization
+Please note that macOS users may need to wrap the GitHub URL in quotes if they encounter issues during installation, as shown in the examples above.
```

Comment:
Now that the actual instruction uses quotes, you don't need this text.

Reply:
Done